app-admin/systemd/autobuild/patches/0002-presets-disable-systemd-networkd-systemd-networkd-wa.patch

```diff
@@ -23,7 +23,11 @@ index da6b9805fd..dcbcd56a7f 100644
 +disable systemd-networkd-wait-online.service
  enable systemd-nsresourced.socket
  enable systemd-pstore.service
- enable systemd-resolved.service
+-enable systemd-resolved.service
++disable systemd-resolved.service
```

Comment:
Why is this patch still around?

Reply:
@OriginCode No consensus was made around this, please revert this patch. Also, move **systemd-networkd** presets to **aosc-os-presets-base** package.

Reply:
@MingcongBai patches have been updated, please rebase this.